raiden/network/transport/matrix/transport.py

```diff
@@ -1053,13 +1060,19 @@ def _get_private_room(self, invitees: List[User]) -> Room:
         self.log.debug("Creating private room", room=room, invitees=invitees)
         return room
 
-    def _get_public_room(self, room_name: str, invitees: List[User]) -> Room:
+    def _get_public_room(self, invitees: List[User]) -> Room:
         """ Obtain a public, canonically named (if possible) room and invite peers """
+        # retrieve address from invitees implicitly, is constant for all invitees
+        assert self._raiden_service is not None  # Extra assertion for mypy
```

Comment:
Please add a message to this assert, use the `RaidenService` as a reference if you need ideas on what to use as a message.